src/rtc_peer.ts

```diff
@@ -183,16 +194,16 @@ export class RTCPeer extends EventEmitter {
             break;
         case 'offer':
             await this.pc.setRemoteDescription(msg);
+            if (this.candidates.length > 0) {
+                this.flushICECandidates();
+            }
```

Comment:
So before we weren't `addIceCandidates`ing here (offer), but now we are?

Reply:
@cpoile Yeah. Candidates need to go after the first call to `setRemoteDescription` call. This way we flush on whichever happens first (either on offer or answer).

Reply:
So was it not working correctly before?

Reply:
@cpoile It was working fine, just sending redundant candidates. There will always be an offer/answer exchange to establish the data channel. I think the remote offer first case may not even be possible given the client is the initiator. But it shouldn't matter in terms of when we can add candidates.